aquacomputer_d5next.c

```diff
@@ -65,26 +66,62 @@ static u8 secondary_ctrl_report[] = {
 #define AQC_FAN_POWER_OFFSET		0x06
 #define AQC_FAN_SPEED_OFFSET		0x08
 
+#define NUM_CTRL_CURVE_POINTS 0x10
+
+#define AQC_FAN_CTRL_PWM_OFFSET 		0x01
+#define AQC_FAN_CTRL_SRC_OFFSET 		0x03
+#define AQC_FAN_CTRL_PID_OFFSET 		0x05
+#define AQC_FAN_CTRL_CURVE_START_OFFSET 	0x13
+#define AQC_FAN_CTRL_CURVE_TEMPS_OFFSET 	0x15
+#define AQC_FAN_CTRL_CURVE_PWMS_OFFSET 		0x25
```

Comment:
The pwm values start from 0x35. Should be:
#define AQC_FAN_CTRL_CURVE_PWMS_OFFSET 		0x35

Reply:
Are you sure? I'm reading values that seem to be correct on my D5 with this offset.

Reply:
fixed in c5a55b66

can check with `for x in $(ls hwmon4/pwm2_auto_point*_pwm); do printf "%s->%s\n" $x $(cat $x); done`